roles/mysql_hardening/README.md

```diff
@@ -12,74 +12,128 @@ It configures:
 - Removes anonymous users, users without a password or authentication_string and test databases
 - various hardening options inside MySQL
 
+Changes of options `log_error` or `datadir` in `mysql_hardening_options` will not be checked for correct permissions. Please change/set `log_error` or `datadir` with the installation role of MySQL before running this role, or you can run this role twice.
+
+Further information is available at [Deutsche Telekom (German)](http://www.telekom.com/static/-/155996/7/technische-sicherheitsanforderungen-si) and [Symantec](http://www.symantec.com/connect/articles/securing-mysql-step-step)
+
 ## Requirements
 
-- Ansible 2.9.0
-- An existing MySQL installation
+- An existing installation of MySQL or MariaDB.
 - python-jmespath on the ansible host
 
-### Example playbook
+<!-- BEGIN_ANSIBLE_DOCS -->
+## Supported Operating Systems
+| Platform | Versions |
+| -------- | -------- |
+| EL | 7, 8, 9 |
+| Ubuntu | bionic, focal, jammy |
+| Debian | bullseye, buster |
+| Amazon |  |
+| opensuse |  |
 
-```yml
-- hosts: localhost
-  collections:
-    - devsec.hardening
-  roles:
-    - mysql_hardening
-```
+## Role Variables
```

Comment:
should we sort this, while we are updating the syntax?

Reply:
Good idea, let me check how to do it. 

This list ist autogenerated. I'll need to add that to the readme, too.

Reply:
they're sorted now.